packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -627,6 +627,27 @@ export class DataStores implements IDisposable {
         }
     }
 
+    /**
+     * When running GC in test mode, this is called to delete objects whose routes are unused. This enables testing
+     * scenarios with accessing deleted content.
+     * @param unusedRoutes - The routes that are unused in all data stores in this Container.
+     */
+    public tombstoneUnusedRoutes(unusedRoutes: string[]) {
+        for (const route of unusedRoutes) {
+            const pathParts = route.split("/");
+            // Delete data store only if its route (/datastoreId) is in unusedRoutes. We don't want to delete a data
+            // store based on its DDS being unused.
+            if (pathParts.length > 2) {
+                continue;
+            }
+            const dataStoreId = pathParts[1];
+            const dataStore = this.contexts.get(dataStoreId);
+            assert(dataStore !== undefined, "No data store retrieved with specified id");
```

Comment:
Any chance this will bite us later once Sweep actually deletes stuff? Maybe just return/no-op in this case rather than asserting?  I like failing fast but not if this case will one day be legit and we should start thinking about forward-compat.

Reply:
When we are actually sweeping, we should not be tombstoning, so this behavior should not be biting us in the back.